packages/matchbox/src/components/Page/Page.js

```diff
@@ -11,16 +11,17 @@ import { Popover } from '../Popover';
 import { UnstyledLink } from '../UnstyledLink';
 import { ScreenReaderOnly } from '../ScreenReaderOnly';
 import { filterByVisible } from '../../helpers/array';
+import { tokens } from '@sparkpost/design-tokens';
 
 const StyledLink = styled(UnstyledLink)`
   text-decoration: none;
 `;
 
 function Breadcrumb({ content, ...rest }) {
   return (
-    <StyledLink {...rest} fontSize="200" lineHeight="200" fontWeight="medium">
-      <Box as="span" display="inline-flex" align-items="center">
-        <ChevronLeft size={20} />
+    <StyledLink {...rest} fontSize="200" lineHeight="450" fontWeight="medium">
+      <Box as="span" display="inline-flex" align-items="center" lineHeight={tokens.spacing_450}>
```

Comment:
What's weird is storybook is applying this lineHeight, but in the App, this lineHeight is missing, which is causing the misalignment.

Not really sure what the fix is here

Reply:
Take a look here for an example: https://matchbox-storybook.netlify.app/?path=/story/layout-page--basic-example

Reply:
I _think_ all we need to do is make `align-items` -> `alignItems`? But still not sure why the bug isn't happening locally